admin/worker/worker.go

```diff
@@ -111,7 +113,7 @@ func (w *Worker) scheduleCron(ctx context.Context, name string, fn func(context.
 		case <-time.After(waitDuration):
 			err := w.runJob(ctx, name, fn)
 			if err != nil {
-				return err
+				w.logger.Error("Failed to run the cronjob", zap.String("cronjob_name", name), zap.Error(err))
```

Comment:
The reason to update the code here is:
When the autoscaling work fails, I think it better to AVOID exiting the admin process.

---

Assuming the unavailability of autoscaler is:
- not harmful to the main process
- can be temporarily unavailable.

Reply:
Seems reasonable – can you make the same change in the old `func (w *Worker) schedule` function?

Reply:
Sure thing!